tests/run_bf.sh

```diff
@@ -15,4 +16,14 @@ else
 fi
 export PYTHONPATH
 
-exec bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json -n prplmesh_docker -x test_flows
+bft -c "${bf_plugins_dir}"/boardfarm_prplmesh/prplmesh_config.json \
+        -n prplmesh_docker -x test_flows -o "${resultdir}" \
+	|| exit $?
+
+failed_test_count=$(jq '.tests_fail' "${resultdir}"/test_results.json)
+re='^[0-9]+$'
+if ! [[ "$failed_test_count" =~ $re ]]; then
+   echo "Unable to parse failed test count:" "$failed_test_count" \
+   && exit 2
+fi
+exit "$failed_test_count"
```

Comment:
There are two issues with return values:
1) it is hard to tell if there was parse error or if there were exactly 2 failing tests
2) as far as I remember, only values in range 0-255 are guaranteed to be supported.

I suggest using 255 for parse error and 254 for number of failures >=254

Reply:
Oh, that makes sense. I added 2 because boardfarm returns 2 when it has a problem.

Reply:
I also changed the exit code when boardfarm fails to 255 too.

Let's try to use it this way for a while, we can make changes if we think that we need actual exit codes instead of failed test count.